### DISCUSSION TITLE
2 fields for language selection missing explanatory title

### DISCUSSION THREAD

**Discussion by c1972** [thorsten/phpMyFAQ#3034] [Ideas]
I see 2 fields for language selection in the admin area:
1. For the admin area, in the header top right
2. For the frontend, in Configuration, Edit configuration, Language

Unfortunately it doesn't say how/where these two fields affect things.

**Please provide better titles, e.g.:**
1. Backend language (current session)
2. Frontend standard language

Because I only have German posts (and don't plan on having any English ones), there is no language selection in my frontend. Deactivating the frontend language selection via backend, that doesn't exist yet (has to be done in source), or does it?

**If** there was such a option, I **then** would also suggest following within documentation:

> If you plan posts in only **ONE language**, you can **deactivate the frontend language selection** using the **[Field in Config]** field.
> 
> When using this option, **please note**:
> If (with only one language posts) this option is used **AND** a **different language** is selected in configuration under **[Frontend standard language]**, no posts will be displayed in the frontend.

Regards, Christian

PS: The more (unusual) questions are answered (once) in docs and backend, the fewer you have to answer individually.

**Comment by thorsten** [comment]
There's no option for removing the language switcher. I renamed the option in the configuration now from "Language" to "Standard language"
